content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -75,7 +75,34 @@ In [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) and later, you
 
 <!-- todo this can be formatted nicer, and not sure this is the correct TS client syntax -->
 
-Additionally, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API. Specifying `@@id([firstName, lastName], name: "fullName")` will allow you to do `prisma.user.findUnique({where: {fullName:{firstName: "Paul", lastName:"Panther"}}})` instead of `prisma.user.findUnique({where: {firstName_lastName:{firstName: "Paul", lastName:"Panther"}}})`.
+Additionally, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API. Specifying `@@id([firstName, lastName], name: "fullName")` will change the API of the exposed Client to uniquely identify entities.
+
+```prisma
+prisma
+  .user
+  .findUnique({
+    where: {
+      fullName:{
+        firstName: "Paul",
+        lastName:"Panther"
+      }
+    }
+  })
+
+instead of
+
+prisma
+  .user
+  .findUnique({
+    where: {
+      firstName_lastName:{
+        firstName: "Paul",
+        lastName:"Panther"
+      }
+    }
+  })
+
+```
```

Comment:
```suggestion
```prisma highlight=6;add|5;delete
prisma
  .user
  .findUnique({
    where: {
      firstName_lastName: {
      fullName: {
        firstName: "Paul",
        lastName: "Panther"
      }
    }
  })
```

Reply:
Does this look right @molebox? If so, let's merge it and look at the preview in the document.